vsintegration/Vsix/VisualFSharpFull/Source.extension.vsixmanifest

```diff
@@ -7,7 +7,7 @@
     <Description xml:space="preserve">Deploy Visual F# Tools templates to Visual Studio</Description>
     <PackageId>Microsoft.FSharp.VSIX.Full.Core</PackageId>
   </Metadata>
-  <Installation>
+  <Installation Experimental="true">
```

Comment:
Yes.  I've just confirmed locally that adding the MyGet feed to VS enabled me to update my installed F# VSIX to the one I that was made as part of the validation build.

The uninstall story is a little more involved and includes:
1. Uninstall F#, either from the VS installer or from the Extensions and Updates dialog which in turn launches the VS installer.
2.  Re-install F# through the VS installer.

The reason for this is because the installer doesn't understand the difference between "component `VisualFSharp` as installed by the installer into `%programfiles%` and component `VisualFSharp` as installed by the user into `%appdata%`".  I'll be filing an internal bug against the installer team to better handle this situation.